.github/workflows/tests.yml

```diff
@@ -192,6 +195,9 @@ jobs:
           brew audit --skip-style --except=version --tap=homebrew/cask-fonts
           brew audit --skip-style --except=version --tap=homebrew/cask-versions
 
+      - name: Generates formula API
+        run: brew generate-formula-api
```

Comment:
```suggestion
        run: brew generate-cask-api
```

Reply:
@reitermarkus controversial suggestion

(typing is hard 😭)

Reply:
To be even more controversial: `name` also needs to be adjusted.

Reply:
I have begrudgingly made these unreasonable changes